app/components/base/TransferWorkflow.tsx

```diff
@@ -210,20 +229,109 @@ function TransferWorkflow() {
       return;
     }
 
-    if (selectedSender.value === "account") {
-      //  TODO: Write if/else statements for all combinations + add JSON data
-    } else if (selectedSender.value === "vcr") {
-      if (isEqual(setSelectedFundsFlow, fundsFlowCombinations.vcr.vcr4)) {
-        setWebhookJsonData(VCRBanktoCRBank);
-      } else if (isEqual(selectedFundsFlow, fundsFlowCombinations.vcr.vcr6)) {
-        setWebhookJsonData(VCRBanktoVCRBankdata);
-      } else {
+    switch (selectedSender.value) {
```

Comment:
Using Switch/Case instead of If/Else for efficiency. Takes less time to run, and is more readable.